docs/providers/aws/events/sqs.md

```diff
@@ -49,7 +49,7 @@ functions:
 
 ## Setting the BatchSize
 
-For the SQS event integration, you can set the `batchSize`, which effects how many SQS messages can be included in a single Lambda invocation. The default `batchSize` is 10, and the max `batchSize` is 10.
+For the SQS event integration, you can set the `batchSize`, which effects how many SQS messages can be included in a single Lambda invocation. The default `batchSize` is 10, and the max `batchSize` is 10000 for a standard queue, 10 for a FIFO queue. You can also set `maximumBatchingWindowInSeconds` to standard queues to specify the maximum amount of time in seconds to gather records before invoking the function. Read more [here](https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html).
```

Comment:
Let's replace "Read more [here]" with "Check [AWS documentation] for more details"

Reply:
Updated: https://github.com/serverless/serverless/commit/d4f172686d62eb3e3fb8c873bcf1bdb32113eda9